app/views/facilities/_facility_fields.html.haml

```diff
@@ -10,7 +10,16 @@
     %label.legend= f.label :banner_notice
     = f.input :banner_notice, input_html: { class: "editor__simple" }, hint: text(".hints.banner_notice"), label: false
 
-%fieldset.well
+- if SettingsHelper.feature_on?(:facility_tile_list)
+  .well
+    %label= text("facilities.thumbnail.thumbnail")
+    = image_tag (@facility.thumbnail.url), class: 'thumbnail_preview'
+    %label= text("facilities.thumbnail.remove")
+    = f.check_box :remove_thumbnail, {}
```

Comment:
Can we do this with: `= f.input :remove_thumbnail, as: :boolean, label: false, inline_label: text("facilities.thumbnail.remove")`. That'll put the label inline with the checkbox. And there are a few other labels on this page (e.g. `accepts_multi_add`). Can we move the localizations to live with those?